.ci.yaml

```diff
@@ -464,6 +464,70 @@ targets:
           {"dependency": "android_virtual_device", "version": "33"}
         ]
 
+  # Device versions of Android integration tests, run via FTL.
+  # TODO(stuartmorgan): Revisit whether we want these once we have more data
+  # about emulator tests; see https://github.com/flutter/flutter/issues/131429.
```

Comment:
Nit: I think we should be more specific for our future selves. "Revisit if physical device tests are redundant once we have more data about emulator tests;

Reply:
Done.